app/controllers/users.rb

```diff
@@ -0,0 +1,21 @@
+
+get '/users/new' do
+  @user = User.new
+  erb :"users/new"
+end
+
+post '/users' do
+  @user = User.new(:email => params[:email],
+              :password => params[:password],
+              :password_confirmation => params[:password_confirmation],
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.